Update Research Data Management chapter [Ready for Review!]

EstherPlomp opened the pull request:
<!--
Please complete the following sections when you submit your pull request. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. Note that text within html comment tags will not be rendered.
-->
### Summary

Checking and adding resources to the chapter, as well as updating internal linking

Fixes 
#3085
#122
#2907


### List of changes proposed in this PR (pull-request)

* Add internal links
* Add resources listed in #3085
* Fix Twitter links


### What should a reviewer concentrate their feedback on?

<!-- This section is particularly useful if you have a pull request that is still in development. You can guide the reviews to focus on the parts that are ready for their comments. We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

- [ ] Everything looks ok?


### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: @reshamas (data visualisation resources), @Karvovskaya for content, 
- [ ]

Arielle-Bennett commented on book/website/reproducible-research/rdm.md:
```diff
@@ -1,10 +1,10 @@
-name: data-ecosystem
-alt: image of the data ecosystem with private and public data
+name: rdm-storage
+alt: A woman standing in front of a very messy closet. She is looking for data that she generated last year. Behind her a person is watching doubtfully whether she can find it in this mess.
```
```suggestion
alt: A cartoon woman standing in front of a very messy closet. She is looking for data that she generated last year. Behind her a person is watching doubtfully, unsure whether she can find it in this mess.
```

Arielle-Bennett commented on book/website/reproducible-research/rdm.md:
```diff
@@ -27,7 +27,7 @@ The following sections in this handbook provide useful context and complementary
 Data here is used as a generic term to encompass all digital objects.
 RDM is a vital part of enabling reproducible research.
-RDM ensures efficiency in research workflows, and also greater reach and impact, as data become FAIR (Findable, Accessible, Interoperable and Reusable).
+RDM ensures efficiency in research workflows, and also greater reach and impact, as data become {ref}`FAIR <rr-rdm-fair>` FAIR (Findable, Accessible, Interoperable and Reusable).
```
```suggestion
RDM ensures efficiency in research workflows, and also greater reach and impact, as data become {ref}`FAIR <rr-rdm-fair>` (Findable, Accessible, Interoperable and Reusable).
```
duplication of FAIR

Arielle-Bennett commented on book/website/reproducible-research/rdm/rdm-article.md:
```diff
@@ -28,46 +28,56 @@ If you are planning to publish an analysis of your published data, it may be wor
 | Article type|  Cost estimate |
```
The text above this states that the cost information was collected in Feb 2022, is that still the case?

EstherPlomp replied:
Yes, this is not updated

EstherPlomp replied:
July 2023! Great spot :)

Arielle-Bennett commented on book/website/reproducible-research/rdm/rdm-data-curation.md:
````diff
@@ -7,34 +7,49 @@ Data curation promotes the use of data from the point of creation to ensuring th
+```{figure} ../../figures/data-curation.*
+---
+name: data-curation
+alt: An illustration describing the seven steps outlined below of the Data Curation Pipeline. The first step is capture, where a person is trying to catch some bugs. The second one is appraisal, where there is a looking glass trying to review the data. The third one is disposal, where some trash is being tossed out of the pipeline. The fourth step is description, which is showing some data charts ...
````
```suggestion
alt: A cartoon illustration describing the seven steps outlined below of the Data Curation Pipeline. The first step is capture, where a person is trying to catch some bugs. The second one is appraisal, where there is a looking glass trying to review the data. The third one is disposal, where some trash is being tossed out of the pipeline. The fourth step is description, which is showing some data charts. The fifth step is preservation, where these charts are then stored in glass jars. The sixth step is access, where individuals would have access to these jars with data in them. The last step is transformation, where different datasets are being transformed into a new one.
```

Arielle-Bennett commented on book/website/reproducible-research/rdm/rdm-data-curation.md:
```diff
@@ -7,34 +7,49 @@ Data curation promotes the use of data from the point of creation to ensuring th
-In data appraisal, you are required to select the appropriate data by entering, digitizing, transcribing, checking, validating and cleaning the data; in addition to anonymizing the data where necessary, describing the data and managing and storing the data.  
-As you appraise the data you need to adhere to the documented guidance, policies and legal requirements in evaluating research data in a particular organization and selecting the data that requires long term preservation. 
+In data appraisal, you are required to select the appropriate data by entering, digitizing, transcribing, checking, validating and cleaning the data. 
+You may also need to to anonymizing the data where necessary.
```
```suggestion
You may also need to anonymise or pseudoanonymise the data where necessary.
```

Arielle-Bennett commented on book/website/reproducible-research/rdm/rdm-fair.md:
```diff
@@ -47,6 +47,16 @@ It is also important to say that the FAIR principles are aspirational: they do n
+## FAIR principles and accessibility
+
+The Accessible in FAIR is not equal to ensuring that your research objects are accessibles to all users. 
+For this, the term “actually accessible”  has been coined by {cite:ps}`Colon2023accessibility` to refer to data that is "easy to locate, obtain, interpret, use, share, and analyze for everybody, including disabled people."
```
```suggestion
For this, the term “actually accessible” has been coined by {cite:ps}`Colon2023accessibility` to refer to data that is "easy to locate, obtain, interpret, use, share, and analyze for everybody, including disabled people."
```

Arielle-Bennett reviewed (commented):
Did a partial review before realising this isn't quite ready for review? But let me know and I'll come back to it tomorrow! ✨

EstherPlomp commented:
> Did a partial review before realising this isn't quite ready for review as you'd mentioned wanting to finish it on Wednesday? But let me know and I'll come back to it tomorrow! ✨

Thanks @Arielle-Bennett! It is now mostly done, apart from the methods / data visualisation section - hopefully finished tomorrow :)

aleesteele commented on book/website/afterword/glossary.md:
```diff
@@ -129,14 +129,14 @@ CRediT Taxonomy
-Data repository
-  See repository.
+Data Repository
+    A long-lived place on the internet where resources (data, software, publications or anything else) can be stored and accessed. Often Data Repositories provide long term preservation and persistent identifiers for the research objects stored.
```
I might use the term "storage space" as opposed to long-lived, for greater clarity. Also, perhaps Data Repositories should be lowercase?

aleesteele replied:
And is that the only feature? Aren't data repositories much bigger in scale as well?

aleesteele replied:
And what's the difference between a database and a data repository? That might be another definition worth adding!

EstherPlomp replied:
I had another go at the definitions! Thanks for jumping in :)

aleesteele commented on book/website/afterword/glossary.md:
```diff
@@ -486,7 +486,7 @@ repo2docker
 
 Repository
-  *Same as Data or Code Reprository*. A long-lived place on the internet where resources (be they data, software, publications or anything else) can be stored and accessed. This keyword is often shortened to ‘repo’.
+  A place on the internet where resources (data, software, publications or anything else) can be stored and accessed. This keyword is often shortened to ‘repo’. See Data Repository if this place is long-lived.
```
Oh! Now I get why you use this wording! 😅

EstherPlomp replied:
I'm happy to change, but I'm not sure if other people had strong feelings about the wording! :P

aleesteele replied:
It might be good to have a general acknowledgement of how it's broadly used, i.e. "While a repository is broadly defined as a central location where resources are stored can be stored and accessed, in computing this is applied to data, software, publications or anything else. This keyword is often shorted to 'repo'.

aleesteele commented on book/website/reproducible-research/rdm/rdm-data-curation.md:
```diff
@@ -7,34 +7,49 @@ Data curation promotes the use of data from the point of creation to ensuring th
 Data curation also enables you to easily access data sets and information that you need in an organised format, you can easily extract data from whichever repository it is stored. 
```
Should the term 'data sets' or 'datasets' be used? I've seen both conventions - and while I don't have strong feelings about it, it it might be good to standardise. Also, maybe the phrase "where you can easily extra data [...]" could be used for clarity in the second half.

EstherPlomp replied:
Good point - I'll change that to datasets I think because it is easier to read/use it as one word

aleesteele commented on book/website/reproducible-research/rdm/rdm-data-curation.md:
```diff
@@ -7,34 +7,49 @@ Data curation promotes the use of data from the point of creation to ensuring th
-Data appraisal and selection are fundamental stages in the data curation process. 
-In data appraisal, you are required to select the appropriate data by entering, digitizing, transcribing, checking, validating and cleaning the data; in addition to anonymizing the data where necessary, describing the data and managing and storing the data.  
-As you appraise the data you need to adhere to the documented guidance, policies and legal requirements in evaluating research data in a particular organization and selecting the data that requires long term preservation. 
+In data appraisal, you are required to select the appropriate data by entering, digitizing, transcribing, checking, validating and cleaning the data. 
```
For data appraisal? During the data appraisal process?

aleesteele commented on book/website/_toc.yml:
```diff
@@ -118,18 +118,22 @@ parts:
+                file: reproducible-research/rdm/rdm-methods
               - title: Data Curation
                 file: reproducible-research/rdm/rdm-data-curation
+              - title: Data Visualisation
```
I think you're using both 'visualisation' and 'visualization' here!

aleesteele commented on book/website/reproducible-research/rdm/rdm-article.md:
```diff
@@ -12,13 +12,13 @@ The benefit of a Data Article is that your output will be {ref}`peer reviewed<cm
 
 Below you can find some journals that publish data articles or papers.
-The costs information was collected in February 2022. 
+The costs information was collected in July 2023. 
```
It might be worth adding something here about what the cost structure is, and the way it works? I remember this being huge news to me when I first learned about it, and making that explicit would probably be really valuable for a researcher

EstherPlomp replied:
I think that may have to go in a separate chapter about APCs!

acocac commented on book/website/reproducible-research/rdm/rdm-resources.md:
```diff
@@ -11,23 +11,60 @@ If you have not read the chapters on {ref}`rr-open` and {ref}`cm-citable` yet, y
 - [An overview of storage solutions and their advantages and disadvantages](https://web.archive.org/web/20180605213316/http://datasupport.researchdata.nl/en/start-the-course/iii-the-research-phase/storing-data/)
+- [Reproducible research](https://coderefinery.github.io/reproducible-research/) by Code Refinery
+- [UKRN Open Research Primers](https://www.ukrn.org/primers/)
 - {cite:ps}`Henry2021RDM`
```
```suggestion
- [Eight principles of good data management](https://doi.org/10.31234/osf.io/5tmfe) by {cite:ps}`Henry2021RDM`
```

acocac commented on book/website/reproducible-research/rdm/rdm-resources.md:
```diff
@@ -11,23 +11,60 @@ If you have not read the chapters on {ref}`rr-open` and {ref}`cm-citable` yet, y
+- [Reproducible research](https://coderefinery.github.io/reproducible-research/) by Code Refinery
+- [UKRN Open Research Primers](https://www.ukrn.org/primers/)
 - {cite:ps}`Henry2021RDM`
 - {cite:ps}`Borer2009RDM`
```
```suggestion
- [Some simple guidelines for effective data management](https://doi.org/10.1890/0012-9623-90.2.205) by {cite:ps}`Borer2009RDM`
```

acocac commented on book/website/reproducible-research/rdm/rdm-resources.md:
```diff
@@ -11,23 +11,60 @@ If you have not read the chapters on {ref}`rr-open` and {ref}`cm-citable` yet, y
+- [UKRN Open Research Primers](https://www.ukrn.org/primers/)
 - {cite:ps}`Henry2021RDM`
 - {cite:ps}`Borer2009RDM`
+- {cite:ps}`Barbrow2017librarians`
```
```suggestion
- [Research data management and services](https://doi.org/10.5860/crln.78.5.274) by {cite:ps}`Barbrow2017librarians`
```

acocac commented on book/website/reproducible-research/rdm/rdm-spreadsheets.md:
```diff
@@ -4,9 +4,8 @@
-There is a collection of [horror-stories](http://www.eusprig.org/horror-stories.htm) that tells how the use of spreadsheets can ruin analysis-based studies due to unexpected behaviour of the spreadsheet or error-prone
-editing processes.
-Some of these mishaps are not unique to spreadsheets, but many, such as [this](https://doi.org/10.1186/s13059-016-1044-7) and [this](https://doi.org/10.1186/1471-2105-5-80), are.
+There is a collection of [horror-stories](https://eusprig.org/research-info/horror-stories/) that tells how the use of spreadsheets can ruin analysis-based studies due to unexpected behaviour of the spreadsheet or error-prone editing processes.
```
We can mention [the embarassing data loss of COVID19 records in England due to bad use of excel](https://www.bbc.co.uk/news/technology-54423988).

EstherPlomp replied:
Oof... I'll add that one - thanks!

acocac commented on book/website/reproducible-research/rdm/rdm-spreadsheets.md:
```diff
@@ -105,10 +106,35 @@ In the English versions, a dot is used since the comma has no meaning (`9,000` w
 
-In particular, only copy-paste from one spreadsheet to another if the process is used very rarely. 
-It is now effortless to read and combine different spreadsheets in the analysis software, with the additional advantage that the software will return an error message if the headers do not fit.
+OpenRefine can be used for tabular data( for example in [social sciences](https://datacarpentry.org/openrefine-socialsci/), [ecology](https://datacarpentry.org/OpenRefine-ecology-lesson/) and [history](https://programminghistorian.org/en/lessons/cleaning-data-with-openrefine).
```
```suggestion
OpenRefine can be used for tabular data (for example in [social sciences](https://datacarpentry.org/openrefine-socialsci/), [ecology](https://datacarpentry.org/OpenRefine-ecology-lesson/) and [history](https://programminghistorian.org/en/lessons/cleaning-data-with-openrefine).
```

acocac commented on book/website/reproducible-research/rdm/rdm-spreadsheets.md:
```diff
@@ -105,10 +106,35 @@ In the English versions, a dot is used since the comma has no meaning (`9,000` w
+
+Comma- or Tab-Separated Value (CSV/TSV) formats are not only best for preservation, but for accessibility as well. 
+For more information:
+- ([Data Curation Primer](https://github.com/DataCurationNetwork/data-primers/blob/master/Accessibility%20Data%20Curation%20Primer/accessibility-data-curation-primer.md#tabular))
```
```suggestion
- [Data Curation Primer](https://github.com/DataCurationNetwork/data-primers/blob/master/Accessibility%20Data%20Curation%20Primer/accessibility-data-curation-primer.md#tabular)
```

acocac commented on book/website/reproducible-research/rdm/rdm-spreadsheets.md:
```diff
@@ -204,5 +228,7 @@ If you work in a team, you should take particular care of the conventions and ma
 
 To learn more about data organisation in spreadsheets, you may have a look at the Data Carpentry lessons for [Social Scientists](https://datacarpentry.org/spreadsheets-socialsci/) and [Ecologists](https://datacarpentry.org/spreadsheet-ecology-lesson/).
 
+To read about recommended pratices, see {cite:ps}`Broman2018data`
```
```suggestion
To read about recommended practices, see {cite:ps}`Broman2018data`
```

acocac commented on book/website/reproducible-research/rdm/rdm-visualisation.md:
```diff
@@ -0,0 +1,54 @@
+* Extract valuable information and patterns from data
+* Communicate this in a clear and comprehensible visual representation
+* Enhance the communication of research findings, making it accessible to a broader audience
+* Identify data quality issues, as outliers and data errors may become more visible in visualisations. 
```
```suggestion
* Identify data quality issues, as outliers and data errors may become more visible in visualisations
```

For consistency with the items above.

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-methods.md:
```diff
@@ -0,0 +1,71 @@
+In both cases the lab notes should record what you did or observed in the laboratory. 
+It should be recording why steps were taken, including mistakes and thoughts or difficulties experienced during data collection and processing. 
+You should take notes in a manner that someone else, with a similar research background or yourself in six months, could use your notebook and repeat the work with the same results. 
+It is very important to clearly describe every step taken and to be specific, otherwise reusers might misinterpret your documentation which will lead to challenges (as can be seen in this [exact instruction challenge video](https://www.youtube.com/watch?v=Ct-lOOUqmyY)). 
```
Strange, I must have added the wrong link! Thanks!

EstherPlomp replied:
I should have added the right link now!

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-methods.md:
```diff
@@ -0,0 +1,71 @@
+
+See [methods & protocols - ReproducibiliTeach](https://www.youtube.com/watch?v=CzpY4A5G70s&list=PLWb8IFSVeQ620plPweZIQSGQODpGOww8r&index=3) for the differences between protocol journals and protocol repositories (at 23:33).
+
+You can also choose to [share your electronic lab notes openly](https://web.archive.org/web/20210309195308/https://www.lornecampbell.org/?p=179). 
```
It is working, just takes a very long time to load sometimes! I think thats a webarchive thing if the server is overloaded

EstherPlomp commented:
> @EstherPlomp I went through all changes and suggested some changes.
> 
> The new and content about methods/protocols and repositories are amazing and pertinent! I've learnt a lot from them (:
> 
> Happy to review the PR again if necessary.

Thanks so much @acocac ! I added all your changes, the resources you forwarded and I fixed the broken YouTube link! Was there anything else that needed to be addressed? Please let me know!

SaranjeetKaur commented on book/website/reproducible-research/rdm/rdm-fair.md:
```diff
@@ -47,6 +47,16 @@ It is also important to say that the FAIR principles are aspirational: they do n
+## FAIR principles and accessibility
+
+The Accessible in FAIR is not equal to ensuring that your research objects are accessibles to all users. 
+For this, the term “actually accessible” has been coined by {cite:ps}`Colon2023accessibility` to refer to data that is "easy to locate, obtain, interpret, use, share, and analyze for everybody, including disabled people."
```
TIL about the term "actually accessible"!

SaranjeetKaur commented on book/website/reproducible-research/rdm/rdm-methods.md:
```diff
@@ -0,0 +1,71 @@
+* **Costs**: Most ELN solutions can only be used through paid plans, or the free plans have reduced functionalities in the number of users, storage space or file sizes.
+Your lab may also not have access to tables or pens that would make the use of ELNs easier.
+Check if your institution is offering a solution that you can use without additional costs.
+* **File format**: always check if files can be exported in your preferred file formats to avoid format lock-in.
```
```suggestion
* **File format**: Always check if files can be exported in your preferred file formats to avoid format lock-in.
```

SaranjeetKaur commented on book/website/reproducible-research/rdm/rdm-methods.md:
```diff
@@ -0,0 +1,71 @@
+* **Sustainability**: Choose a solution that has a larger chance of being around for a long time.
+* **Drawing**: Not all solutions have the tools or capabilities to include drawings or integration with drawing software.
+* **Security**: Before using an ELN solution you should check any backup plans and data security measures, especially if you're working with sensitive data.
+* **Laboratory situation**: Your lab may not have an internet collection or electronic tools could affect samples, reagents or magnets in the lab. 
```
```suggestion
* **Laboratory situation**: Your lab may not have an internet collection or electronic tools could affect samples, reagents or magnets in the lab. 
```

SaranjeetKaur commented on book/website/reproducible-research/rdm/rdm-repository.md:
```diff
@@ -0,0 +1,159 @@
+Supplemental materials are not following the FAIR principles - as there is no seperate DOI assigned to the supplemental materials which makes it difficult to retrieve these materials. 
+Next to supplemental materials not being aligned with the FAIR principles, there are other reasons why a data repository is a better solution: 
+
+- Data control: supplementary materials cannot be updated, unlike materials available at data repositories.
```
```suggestion
- Data control: Supplementary materials cannot be updated, unlike materials available at data repositories.
```

SaranjeetKaur commented on book/website/reproducible-research/rdm/rdm-repository.md:
```diff
@@ -0,0 +1,159 @@
+
+- Data control: supplementary materials cannot be updated, unlike materials available at data repositories.
+- Interoperability: If publishers only allow text and PDF formats it hampers data sharing and it will be difficult to reuse the data.
+- Availability: supplementary materials are difficult to access if the article is behind the paywall, and links to supplementary materials can break (since they do not have their own persistent identifier).
```
```suggestion
- Availability: Supplementary materials are difficult to access if the article is behind the paywall, and links to supplementary materials can break (since they do not have their own persistent identifier).
```

SaranjeetKaur commented on book/website/reproducible-research/rdm/rdm-repository.md:
```diff
@@ -0,0 +1,159 @@
+It is possible to add more metadata into the project Wiki or submit in form of a separate file.
+- It is possible to add license to a project to specify how others are allowed to copy, distribute, and make use of this work. 
+- OSF provides detailed landing pages to document projects.
+Each *component* has its own wiki that allows to add reach documention on multiple levels.
```
```suggestion
Each *component* has its own wiki that allows to add reach documentation on multiple levels.
```

acocac commented:
> Thanks so much @acocac ! I added all your changes, the resources you forwarded and I fixed the broken YouTube link! Was there anything else that needed to be addressed? Please let me know!

@EstherPlomp thanks for addressing my suggestions. Everything looks good to me. I think it'd be great to check existing or potential new Scriberia illustrations that could be added to the methods / data visualisation sections.

EstherPlomp commented:
@all-contributors please add @Karvovskaya for content

EstherPlomp commented:
@all-contributors please add @reshamas for content
